src/backend/utils/vkbasalt.py

```diff
@@ -0,0 +1,257 @@
+# vkbasalt.py: library supplying the logics and functions to generate configs
+#
+# Copyright 2022 vkbasalt-cli Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from os import path, environ, system, remove
+from sys import exit
+from shutil import copyfile
+import configparser
+import logging
+
+
+def Parse(args):
+    # Apply default settings if possible
+    if args.default:
+        install_paths = [
+            "/usr/lib/extensions/vulkan/vkBasalt/etc/vkBasalt",
+            "/usr/local",
+            "/usr/share/vkBasalt",
+        ]
+        for i in range(len(install_paths)):
```

Comment:
ditto. If you need the index do `for index, item  in enumerate(list)`

Reply:
Why use this over the one I have right now? I don't think I'll use `index`.